composer.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "alleyinteractive/options-importer",
+  "type": "wordpress-plugin",
+  "keywords": ["wordpress", "plugin"],
+  "authors": [
+    {
+      "name": "Alley Interactive",
+      "email": "noreply@alleyinteractive.com"
+    }
+  ],
+  "require": {
+    "composer/installers": "~1.0"
+  },
+  "require-dev": {
```

Comment:
Not a blocker, but this would be a good opportunity to add the Alley Coding Standards. See https://github.com/alleyinteractive/Alley-Coding-Standards for setup instructions. ACS automatically sets up PHPCS, WPCS, and VIPWPCS for you.

Reply:
We need certain things like array long syntax to support older versions of PHP, so for now it might be easier to use vanilla wpcs.

Reply:
Short array syntax is available since WP 5.4, so if we drop support for 5.3 then we should be fine. Either way, happy to defer this.